wpilibjExamples/build.gradle

```diff
@@ -185,6 +185,8 @@ model {
                                     testTask.useJUnitPlatform()
                                     testTask.filter {
                                         includeTestsMatching("edu.wpi.first.wpilibj.examples.${entry.foldername}.*")
+                                        // armsimulation regularly fails on CI Win64Debug
+                                        excludeTestsMatching("edu.wpi.first.wpilibj.examples.armsimulation.*")
```

Comment:
It would be better imo to tag the test with `@DisabledOnOs(Os.Windows)` (or whatever that annotation is) rather than disabling it from here.

Reply:
The problem is that this is an example, and we don't want to disable the test for users.

Reply:
If it fails on CI, won't it fail on team windows machines too?
Besides, I'm not sure the tests are copied to example-based projects.

Reply:
Not in general.  This appears to only fail sometimes on CI machines in debug mode, so is highly likely to be a timing-related failure that would be extremely unlikely to fail on a team machine.

Reply:
It's a good question though whether the test folder is distributed.  If it's not then we could just do the DisabledOnOs thing.

Reply:
I made a armsimulation example project in Alpha 1 VS Code and there's no test folder.

I thought there was an issue to copy the tests to user projects already open, but I didn't see it

Reply:
>  Besides, I'm not sure the tests are copied to example-based projects.

If this is the case, we should fix that, considering we have a UnitTest example.